blog_system/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path, include
+from debug_toolbar.toolbar import debug_toolbar_urls
```

Comment:
The import statement `from debug_toolbar.toolbar import debug_toolbar_urls` is not correct. The debug_toolbar_urls should be imported from debug_toolbar, like this: `from debug_toolbar import urls as debug_toolbar_urls`.